app/src/main/java/com/dessalines/thumbkey/keyboards/CommonKeys.kt

```diff
@@ -47,20 +60,69 @@ val NUMERIC_KEY_ITEM =
         swipeType = SwipeNWay.EIGHT_WAY,
         swipes = mapOf(
             SwipeDirection.TOP to KeyC(
-                display = KeyDisplay.IconDisplay(Icons.Outlined.SelectAll),
-                action = KeyAction.SelectAndCopyAll,
+                display = KeyDisplay.IconDisplay(Icons.Outlined.ContentCopy),
+                action = KeyAction.Copy,
                 color = ColorVariant.MUTED,
             ),
             SwipeDirection.TOP_LEFT to KeyC(
-                display = KeyDisplay.IconDisplay(Icons.Outlined.Keyboard),
-                action = KeyAction.SwitchIME,
+                display = KeyDisplay.IconDisplay(Icons.Outlined.SelectAll),
+                action = KeyAction.SelectAll,
+                color = ColorVariant.MUTED,
+            ),
+            SwipeDirection.TOP_RIGHT to KeyC(
+                display = KeyDisplay.IconDisplay(Icons.Outlined.ContentCut),
+                action = KeyAction.Cut,
+                color = ColorVariant.MUTED,
+            ),
+            SwipeDirection.LEFT to KeyC(
+                // Undo
```

Comment:
You can remove these two `// Undo` and Redo comments, they're unecessary once you extract them into KeyActions.